### PR TITLE
prepare v0.3.0

### PR DESCRIPTION
**Pull request by eguzki**
Diff 0.2.2 and 0.3.0

```
$ diff manifests/0.2.2/apicast-community-operator.v0.2.2.clusterserviceversion.yaml manifests/0.3.0/apicast-community-operator.v0.3.0.clusterserviceversion.yaml 
57c57
<   name: apicast-community-operator.v0.2.2
---
>   name: apicast-community-operator.v0.3.0
60c60
<   replaces: apicast-community-operator.v0.2.1
---
>   replaces: apicast-community-operator.v0.2.2
81c81
<     See [the upgrade guide](https://github.com/3scale/apicast-operator/blob/v3.8/doc/operator-user-guide.md#upgrading-APIcast) for more information.
---
>     See [the upgrade guide](https://github.com/3scale/apicast-operator/blob/v3.9/doc/operator-user-guide.md#upgrading-APIcast) for more information.
84c84
<     Documentation can be found on our [website](https://github.com/3scale/apicast-operator/tree/v3.8).
---
>     Documentation can be found on our [website](https://github.com/3scale/apicast-operator/tree/v3.9).
135,137c135,136
<                   value: "quay.io/3scale/apicast:v3.8.0"
<                 image: quay.io/3scale/apicast-operator:v0.2.0
<                 imagePullPolicy: Always
---
>                   value: "quay.io/3scale/apicast:v3.9.0"
>                 image: quay.io/3scale/apicast-operator:v0.3.0
268c267
<   version: 0.2.2
---
>   version: 0.3.0
```

### CODE REVIEW

**Comment by miguelsorianod** [comment]
I wonder if we should name the new CSV `0.3.1` instead of `0.3.0`?

In `0.2.x` case we started creating `0.2.1` (instead of `0.2.0`) because of a similar case.
However, it seems clear at this point in time that the CSV versions and replaces between product metadata and community metadata seem to be different. This is, the release lifecycle seems to be independent.

**Comment by eguzki** [comment]
For me this is the first 0.3.x release, and therefore it should be 0.3.0.

**Comment by miguelsorianod** [comment]
As talked let's go forward with this taking into account:
* We now have different release lifecycle and versioning between community and product. We should not confuse CSV versions. Even if they have the same version number they are independent and different.
* If N CSV product releases contain different upgrade code and we decide to join them as 1 CSV communtity release should think about the implications at upgrade level for community (maybe merging upgrade code ...)

**Review by miguelsorianod — APPROVED** [review]

File: manifests/0.3.0/apicast-community-operator.v0.3.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+      ]
+    capabilities: Full Lifecycle
+    categories: Integration & Delivery
+    certified: "false"
+    containerImage: quay.io/3scale/apicast-operator:v0.2.0
+    createdAt: "2019-10-27T22:40:00Z"
+    description: APIcast is an API gateway built on top of NGINX. It is part of the 3scale API Management Platform
+    repository: https://github.com/3scale/apicast-operator
+    support: Red Hat, Inc.
+  name: apicast-community-operator.v0.3.0
```
Comment (eguzki) [inline manifests/0.3.0/apicast-community-operator.v0.3.0.clusterserviceversion.yaml:57]: :eyes:

**Pull request merged**